src/nextjs/index.ts

```diff
@@ -159,6 +166,11 @@ export class OttofellerNextjsProject extends NextJsTypeScriptProject implements
     PullRequestTest.addToProject(this, options)
     ProjenDriftCheckWorkflow.addToProject(this, options)
 
+    // ANCHOR Code owners
+    if (this.github && options.codeOwners) {
+      new CodeOwners(this.github, {codeOwners: options.codeOwners})
```

Comment:
@quesabe on a side note, we repeat this and other parts of the method in almost all projen projects. Is it worth to do a refactoring in one of the following PRs and move repetitive actions in its own construct?

Reply:
I thought about it - for me a singe if-statement didn't seem like a lot of repetition. So I decided to leave it in the present form.

However if you think it's better to have it as a static method `addToProject` (similar to other contracts), I can do it. It's not a lot of work, so I can incorporate the change in the current PR.

BTW. I though about trying to commit a similar construct into `projen` repo. And later if they accept it, use the one from `projen`, getting rid of our own. Though it still should not affect our own solution - so we can adopt any form we'd like.

Reply:
Proposing an update to projen sounds like a great idea. In general my suggestion was about moving all these methods/constructs calls which are common for all projects to a single function. This should eliminate repetition and probably save us some time in the future.

Reply:
Makes sense. I've refactored into a static method and in the project we have just the following:
```typescript
CodeOwners.addToProject(this, options)
```